PetScanList/one_page_bot.py

```
@@ -36,62 +40,67 @@ def is_petscan_list_page(page_title):
     return "petscan list" in page_title.lower().strip().replace("_", " ")
 
 
-def update_page_content(page_title):
+def update_page_content(page_title, wiki):
     """
     Update the content of a Wikipedia page using the `text_bot.process_text` function.
     """
+    # ---
+    result_class = ""
```

Ruff (0.8.2)

Local variable `result_class` is assigned to but never used

Remove assignment to unused variable `result_class`

(F841)